examples/rcbd_one_var.R

```diff
@@ -0,0 +1,109 @@
+# This is an example of a single variate RCBD.
+#
+# To run the script type:
+# source('rcbd_one_var.R', print.eval = TRUE)
+#
+# The code that will be displayed to the user is bounded by this separator:
+#-----------------------------------------------------------------------------#
+
+# Load the necessary libraries.
+#-----------------------------------------------------------------------------#
+library(car)  # for leveneTest()
+library(agricolae)  # for LSD.test()
+#-----------------------------------------------------------------------------#
+
+sep <- function(n){
+  # This function simply prints a line of "=" to the screen as a separator and
+  # will not show up in the app. It is only here for nice printing.
+  line <- paste0(paste(replicate(n, "="), collapse = ""), '\n')
+  cat(line)
+}
+
+# Load the data.
+#-----------------------------------------------------------------------------#
+my.data <- read.csv("Wheat_yield_data.csv")
+#-----------------------------------------------------------------------------#
+
+# Set Block and Treatment as factors.
+#-----------------------------------------------------------------------------#
+my.data$Block <- as.factor(my.data$Block)
+my.data$Treatment <- as.factor(my.data$Treatment)
+#-----------------------------------------------------------------------------#
+
+# Construct the model.
+# NOTE : aov() can be used here and the as.factor() calls would be unnecessary.
+#-----------------------------------------------------------------------------#
+model <- lm(Yield ~ Treatment + Block, my.data)
+#-----------------------------------------------------------------------------#
+
+# Print the ANOVA table for the model.
+cat('ANOVA Table\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+anova(model)
+#-----------------------------------------------------------------------------#
+sep(79)
+
+# Show the confidence intervals for the intercept and each virus level.
+cat('Confidence Intervals\n')
+sep(79)
+#-----------------------------------------------------------------------------#
+confint(model)
```

Comment:
Confidence intervals are only needed for the means of the treatments (not blocks).